vim/load.ts

```diff
@@ -0,0 +1,63 @@
+import { Denops } from "../deps.ts";
+import { execute } from "./execute.ts";
+
+interface FileInfo {
+  isDirectory: boolean;
+}
+
+export async function load(
+  denops: Denops,
+  filePathInfo: URL,
+): Promise<void> {
+  const scriptPath = await setFile(denops, filePathInfo);
+
+  await execute(denops, `source ${scriptPath}`);
+}
+
+async function setFile(denops: Denops, filePathInfo: URL): Promise<string> {
+  const re = /.*\//;
+  const homeDir = await Deno.env.get("HOME") as string;
+
+  if (filePathInfo.protocol === "file:") {
```

Comment:
:memo:

`URL.protocol` includes the final `':'`.

https://developer.mozilla.org/en-US/docs/Web/API/URL/protocol